From f46ecb853d9e9fd4a9462fc37b99593c013f6d2d Mon Sep 17 00:00:00 2001
From: Guillem Jover <guillem@debian.org>
Date: Thu, 5 May 2011 03:24:57 +0200
Subject: [PATCH] Update svgalib related entries

Remove svgalib entry, the .dsc already contains information about the
supported architectures.

Remove packages no longer available in archive depending on svgalib.

Signed-off-by: Guillem Jover <guillem@debian.org>
---
 Packages-arch-specific |    8 --------
 1 files changed, 0 insertions(+), 8 deletions(-)

diff --git a/Packages-arch-specific b/Packages-arch-specific
index f420eb2..fddc10d 100644
--- a/Packages-arch-specific
+++ b/Packages-arch-specific
@@ -185,7 +185,6 @@ lilo-installer: i386 amd64 lpia					      # only useful if you have lilo
 %linux-modules-di-s390-2.6: s390
 %linux-modules-di-sparc-2.6: sparc
 %linux-wlan-ng: amd64 i386 powerpc arm armel alpha hppa lpia	      # ANAIS [?]
-lirc-svga: i386							      # ANAIS, #267323
 %lisaac: !alpha !amd64 !ia64 !kfreebsd-amd64                          # no 64-bit support
 lkl: i386 kfreebsd-i386						      # i386 kb hardware specific
 %llvm: !ia64 !mips !mipsel
@@ -334,7 +333,6 @@ silo: sparc							      # sparc lilo
 subterfugue: i386						      # i386 specific process tracing etc.
 %superiotool: !hppa !ia64 !m68k !mips !mipsel !powerpc !sh4 !sparc    # sys/io.h
 spu-tools: powerpc ppc64					      # PowerPC specific
-%svgalib: i386 amd64                                                  # PC-Architecture specific
 svgatextmode: i386 amd64                                              # Hardcoded i386 assembler
 %sysconfig: s390						      # ANAIS s390 specific
 syslinux: i386 amd64 lpia                                             # i386 boot loader thingy
@@ -460,7 +458,6 @@ yaboot-installer: powerpc					      # PowerPC Bootloader installer
 yforth: i386 m68k sparc arm armel powerpc kfreebsd-i386 kfreebsd-amd64 # compiler
 %ypsilon: i386 amd64                                                  # ANAIS
 z88dk: !alpha                                                         # 32bit only (using -m32 on amd64 and ia64)
-zblast-svgalib: i386						      # i386 only per David Pashley
 %zipl-installer: s390						      # installer stuff
 zorroutils: m68k powerpc					      # zorro (Amiga) specific
 zsnes: i386							      # Mostly written in i386 assembler
@@ -519,12 +516,7 @@ isdneurofile: amd64 alpha arm armel i386 ia64 m68k powerpc sparc mips mipsel lpi
 %tvflash: i386 ia64 amd64 powerpc
 
 ## Depend: on svgalib
-bmv: amd64 i386
 dvisvga: amd64 i386
-luxman: amd64 i386
-razzle: amd64 i386
-sabre: amd64 i386
-vgacardgames: amd64 i386
 zgv: amd64 i386
 
 ## Depend: on cmucl
-- 
1.7.5

